lib/yardstick/rule_description/classifier.rb

```diff
@@ -0,0 +1,105 @@
+module Yardstick
+  # Rule description composed of tokens which can be formatted
+  class RuleDescription
+    # Token classification
+    module Classifier
+      # List of classifiers
+      class List
+        extend Forwardable
+        include Concord.new(:classifiers), Enumerable
+
+        # Classify a token by returning the first match
+        #
+        # @param token [String] markup text
+        #
+        # @return [Token]
```

Comment:
Hm this is a call side I didn't expect since my current use case always expects at least one `Default` token which always matches.  I could add an exception after the `each` block to protect against this call side where `classifiers` would be returned.